test/test_autograd.py

```diff
@@ -8743,6 +8743,16 @@ def backward(cls, ctx, gO):
         self.assertEqual(y, y2)
         self.assertEqual(y_expected, y2_expected)
 
+    @unittest.skipIf(not TEST_CUDA, "test requires CUDA")
+    def test_gradcheck_rng_generator_device_placement(self):
```

Comment:
nit:
```suggestion
    def test_gradcheck_default_device_placement_context(self):
```

Add a short comment like: "In fast gradcheck we create random grad vector on cpu first always using a cpu generator, make sure this still works when the default device is set to something else"